lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -684,9 +715,9 @@ migrateManually tr proxy defaultFieldValues =
     -- | This table became @protocol_parameters@.
     removeOldTxParametersTable :: Sqlite.Connection -> IO ()
     removeOldTxParametersTable conn = do
-        dropTable <- Sqlite.prepare conn "DROP TABLE IF EXISTS tx_parameters;"
-        void $ Sqlite.stepConn conn dropTable
-        Sqlite.finalize dropTable
+        dropTable' <- Sqlite.prepare conn "DROP TABLE IF EXISTS tx_parameters;"
+        void $ Sqlite.stepConn conn dropTable'
+        Sqlite.finalize dropTable'
```

Comment:
alternatively, I've started using something more generic for this intermediate variable in other migration like `statement` or `stmt`. 

```
        stmt <- Sqlite.prepare conn "DROP TABLE IF EXISTS tx_parameters;"
        void $ Sqlite.stepConn conn stmt
        Sqlite.finalize stmt
 ```